src/topology/metric_space/thickened_indicator.lean

```diff
@@ -0,0 +1,211 @@
+/-
+Copyright (c) 2022 Kalle Kytölä. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kalle Kytölä
+-/
+import data.real.ennreal
+import topology.continuous_function.bounded
+
+/-!
+# Thickenings and thickened indicators
+
+## Main definitions
+
+## Main results
+
+-/
```

Comment:
Expand?

Reply:
I will!

But I think the contents of the file must be decided before writing the module docstring. Specifically, the docstring will depend on whether thickenings are moved from `hausdorff_distance.lean` to a separate file that includes thickened indicators. The theme of the new file would become quite coherent that way. But the current file structure has fewer imports when one is only using thickenings and not thickened indicators. Which option is better in your opinion?

Reply:
I think the best option is to have three files: one on inf_dist and thickenings, one on hausdorff distance, and one on thickened indicators (because thickened indicators import a good deal on continuous functions, so normed spaces and so on, which are not needed for the other file).. So your PR can go as it is now, and the splitting of the other file can be made later on.

Reply:
Thank you, this plan sounds great to me!